cachecontrol/controller.py

```diff
@@ -90,21 +111,21 @@ def cached_request(self, request):
         Return a cached response if it exists in the cache, otherwise
         return False.
         """
-        cache_url = self.cache_url(request.url)
-        logger.debug('Looking up "%s" in the cache', cache_url)
         cc = self.parse_cache_control(request.headers)
 
         # Bail out if the request insists on fresh data
         if 'no-cache' in cc:
             logger.debug('Request header has "no-cache", cache bypassed')
             return False
 
-        if 'max-age' in cc and cc['max-age'] == 0:
+        if 'max-age' in cc and cc['max-age'] == '0':
```

Comment:
I think this might be nicer as `int(cc['max-age']) == 0`, just in case we actually do get back an integer for some reason